packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -257,6 +249,29 @@ const DEFAULT_TOKEN_PRICE_MAX_CONSECUTIVE_FAILURES =
 
 const DEFAULT_DEGRADED_THRESHOLD = 5_000;
 
+type CryptoData = {
+  allTimeHigh: number;
+  allTimeLow: number;
+  circulatingSupply: number;
```

Comment:
Since the parameter we pass to the API is called `includeMarketData`, should this be called `MarketData` instead of `CryptoData`?

Also, what are your thoughts on adding JSDoc for this type. In particularly, it'd be nice to explain what each property does.

Something like:

``` suggestion
/**
 * The shape of the data that the /spot-prices endpoint returns.
 */
type MarketData = {
  /**
   * The all-time highest price of the token.
   */
  allTimeHigh: number;
  /**
   * The all-time lowest price of the token.
   */
  allTimeLow: number;
  /**
   * The number of tokens in circulation.
   */
  circulatingSupply: number;
```